Makefile

```diff
@@ -132,10 +132,14 @@ docker-images: $(scala-services) $(dockerfile-services)
 # Platform actions
 .PHONY: start stop restart test
 start:
+ifeq ($(RENGA_ENDPOINT),)
+	RENGA_ENDPOINT="http://localhost" RENGA_CONTAINERS_ENDPOINTS=http://`docker network inspect bridge --format='{{(index (index .IPAM.Config) 0).Gateway}}'` $(MAKE) start
+else
```

Comment:
won't this skip the docker-compose commands if `RENGA_ENDPOINT` is defined?

Reply:
If RENGA_ENDPOINT is undefined or empty, it will set RENGA_ENDPOINT and RENGA_CONTAINERS_ENDPOINT to the default values, and recursively invoke the make start command with the defined variables.
If it is defined and not empty (e.g. in the recursive call), it will invoke the docker-compose command.

Reply:
Ah didn't see the recursion, sorry -- I also did this similarly on L43 and L48, not sure which method is preferred?

Reply:
Sorry, I didn't realize that it was part of the change when I resolved the conflicts. Yes you can ignore this PR.